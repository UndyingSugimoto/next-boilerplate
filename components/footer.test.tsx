import { shallow } from 'enzyme';
import React from 'react';
import Footer, { FooterTag } from './footer';
import 'jest-styled-components';

describe('Chart', () => {
  test('レンダリングの確認', () => {
    const wrapper = shallow(<Footer></Footer>);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(FooterTag).length).toBe(1);
  });
});
