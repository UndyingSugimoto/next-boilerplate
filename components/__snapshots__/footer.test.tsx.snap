// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart レンダリングの確認 1`] = `
<styled.footer>
  <div>
    footer-content
  </div>
</styled.footer>
`;
